core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphQlCallResult.kt

```diff
@@ -1,14 +1,17 @@
 package org.evomaster.core.problem.graphql
 
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
 import com.google.common.annotations.VisibleForTesting
 import org.evomaster.core.problem.httpws.HttpWsCallResult
 import org.evomaster.core.search.action.Action
-import org.evomaster.core.search.action.ActionResult
 
 class GraphQlCallResult : HttpWsCallResult {
 
     companion object{
         const val LAST_STATEMENT_WHEN_GQL_ERRORS = "LAST_STATEMENT_WHEN_GQL_ERRORS"
+        private val mapper: ObjectMapper = ObjectMapper()
```

Comment:
no need to specify type `: ObjectMapper` if variable is assigned directly

Reply:
Copy-paste takes a wrong turn :sweat_smile: